Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/detectiontracker/DefaultExposureDetectionTracker.kt

```diff
@@ -134,6 +117,54 @@ class DefaultExposureDetectionTracker @Inject constructor(
         }
     }
 
+    private fun finishExposureDetectionWithoutIdentifier(
+        map: MutableMap<String, TrackedExposureDetection>,
+        result: Result
+    ) {
+        Timber.d(
+            "finishExposureDetectionWithoutIdentifier(): " +
+                "Get identifier of newest unfinished detection or create new identifier"
+        )
+        val identifier = map
+            .map { it.value }
+            .filter { it.finishedAt == null }
+            .maxByOrNull { it.startedAt.millis }
+            ?.identifier ?: kotlin.run {
```

Comment:
A bit difficult to read. How could we make this easier on the :eyes:

Reply:
good point. 
_?.identifier ?: kotlin.run_ is out.
readability improved